lakeshore336/system_tests/tests/lksh336.py

```diff
@@ -0,0 +1,186 @@
+import contextlib
+import unittest
+
+from parameterized import parameterized
+
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import get_default_ioc_dir
+from utils.test_modes import TestModes
+from utils.testing import get_running_lewis_and_ioc, parameterized_list
+
+
+DEVICE_PREFIX = "LKSH336_01"
+
+IOCS = [
+    {
+        "name": DEVICE_PREFIX,
+        "directory": get_default_ioc_dir("LKSH336"),
+        "macros": {},
+        "emulator": "Lksh336",
+    },
+]
+
+TEST_MODES = [TestModes.DEVSIM]
```

Comment:
Can any of the tests run in recim? I notice there are some `SIML`/`SIOL` pvs defined so looks like at least some of them should work?